service/history/api/updateworkflow/api.go

```diff
@@ -257,7 +267,8 @@ func (u *Updater) OnSuccess(
 			// If subsequent attempt succeeds within current context timeout, caller of this API will get a valid response.
 			err = nil
 		}
-	} else {
+	default:
+		// Normal WFT was created.
```

Comment:
I believe there is no such case any more.

Reply:
👍 I filed a separate ticket to address this